src/SeleniumLibrary/keywords/element.py

```diff
@@ -112,21 +113,42 @@ def page_should_contain_element(self, locator, message=None, loglevel='INFO'):
         The ``message`` argument can be used to override the default error
         message.
 
+        The ``limit`` argument can used to define how many elements the
+        page should contain. When ``limit`` is `None` page can contain one or
+        more elements. When limit is a number, page must contain same number
+        of elements.
+
         See `Page Should Contain` for explanation about the ``loglevel``
         argument.
+
+        The ``limit`` argument is new in SeleniumLibrary 3.0
```

Comment:
`.` missing at the end. I'd also move this note at the very end of the doc after the examples.